src/main/java/levy/gameoflife/RleParser.java

```diff
@@ -0,0 +1,74 @@
+package levy.gameoflife;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class RleParser {
+    public int getWidth() {
+        return width;
+    }
+
+    public int getHeight() {
+        return height;
+    }
+
+    private int width;
+    private int height;
+    private int[][] grid;
+
+    public int[][] parse(String rleContent) {
+        String[] lines = rleContent.split("\n");
+        Pattern headerPattern = Pattern.compile("x\\s*=\\s*(\\d+),\\s*y\\s*=\\s*(\\d+)");
```

Comment:
What does this do?

Reply:
this looks for this format( x = <number>, y = <number>) to find the dimensions